### PR TITLE
Update handler-slack.rb

### PR DESCRIPTION
**Pull request by jonguijarro**
## Pull Request Checklist

**Is this in reference to an existing issue?**
No
#### General

- [x] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [x] Existing tests pass 

#### New Plugins

- [ ] Tests

- [x] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
Update handler-slack to show uchiwa links on slack message
#### Known Compatablity Issues
Works with uchiwa dashboard

### CODE REVIEW

**Comment by warmfusion** [comment]
You need to update the readme with information on how the new dashboard config works, including updating the example. As well as the changelog with your new feature.

**Comment by jonguijarro** [comment]
image

In this case sensu02/sensu-server links to our dashboard
http://sensu.dashboard/#/client/sensu/sensu02?check=sensu-server
where  http://sensu.dashboard/#/client/sensu/ was defined as param dashboard on the handler configuration.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.4.0] - 2017-08-07
```
Comment (majormoses) [inline CHANGELOG.md]: 1.4.0 was already released. Anyways it is preferable that you leave off the versioning to the maintainers as there is no guarantee on merge order or how long a review will take.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-slack/versions/1.5.0

**Pull request merged**